package.json

```diff
@@ -178,6 +178,17 @@
           "description": "Determines whether to show the Quarkus extension descriptions when selecting Quarkus extensions.",
           "scope": "window"
         },
+        "quarkus.tools.forceToQuarkusLanguage": {
+          "type": "string",
+          "enum": [
+            "never",
+            "always",
+            "prompt"
+          ],
```

Comment:
"enumDescriptions": [
            "ignore language mismatch for Quarkus properties",
            "Automatically correct the language for mismatched Quarkus properties",
            "Prompt correcting the language for mismatched Quarkus properties"
          ]

Reply:
fixed

Reply:
fixed